robust/linearize_twoterm_posynomials.py

```diff
@@ -40,7 +40,7 @@ def intersection_point_func(x, a, b, eps):
         return a * x + b - np.log(1 + np.exp(x)) + eps
 
     @staticmethod
-    def iterate_two_term_posynomial_linearization_coeff(r, eps):
+    def iterate_linearization_coeff(r, eps):
```

Comment:
seems like removing `two_term` from all these function names obfuscates things a bit... is it documented somewhere that the methods are two-term?

Reply:
These methods are always called on the class (eg. TwoTermPosynomial.iterate_linearization_coeff), not usually an instance of the class, so no obfuscation here. Also, I don't really want to deal with a gazillion pint warnings in the future due to lines being too long.

Reply:
gotcha, makes sense. thanks for explanation.